### DISCUSSION TITLE
Non-coders are dealing with tons of code being added to notes when changing editors, easy 1-button click fix needed

### DISCUSSION THREAD

**Discussion by initiator1** [standardnotes/forum#2930] [General]
Ladies and gentlemen,

Thank you so much for everything that makes up the community and leadership around this application.

Please send a patch update with some simple option for non-coders that will remove all the code that switching around between text editors (between Android and Windows platforms on one account) has caused me so many many many many, way too many multiple times over the years.

It is such an incredible problem that if you are able to solve will prove your company's worth. This should never, EVER be happening to anyone. Your team has always been a huge inspiration based on the original TED talk that brought me to your company. This is a make-or-break level priority. Right now, and for many years, things have been broken. Please take this problem absolutely seriously! 

Sorry for not bringing attention to it sooner. I would have loved it to have been solved a long time ago.

Many thanks in advance. Don't let everyone down!

Best regards.

**Comment by shompoe** [comment]
I have a tip for you which may or may not help. Try selecting all the text from within your "rich text" notes then "paste as plain text" into a new note or in place in the same note after switching editors. This should remove all the formatting code.

The solution which you are asking for (a one button fix) is also in the new supernote editor, but that editor is still a work in progress.
